plugin/trino-mongodb/src/main/java/io/trino/plugin/mongodb/MongoSession.java

```diff
@@ -206,7 +206,7 @@ public void createTable(SchemaTableName name, List<MongoColumnHandle> columns, O
             throw new SchemaNotFoundException(name.getSchemaName());
         }
         createTableMetadata(name, columns, comment);
-        // collection is created implicitly
```

Comment:
why this change?

Reply:
The commit message has
> Let MongoDB creates a collection explicitly because 
implicit creation doesn't happen in case of long identifiers.

Reply:
Does `createTableMetadata(name, columns, comment` succeed?
does it create any resources?